src/core/network/createNetwork.js

```diff
@@ -31,7 +31,7 @@ export default (config, logger, lifecycle, networkStrategy) => {
 
     const response = createResponse(parsedBody);
 
-    return lifecycle.onResponse({ response }).then(() => response);
+    return lifecycle.onResponse({ response, requestId }).then(() => response);
```

Comment:
Jon, the response already contains the request ID that was provided, or the one that was created by Konductor if none was provided. Just an FYI, I don’t think it hurts to have it to match the onResponseError.